crates/router/tests/connectors/tsys.rs

```diff
@@ -286,9 +296,14 @@ async fn should_refund_succeeded_payment_multiple_times() {
 #[actix_web::test]
 async fn should_sync_refund() {
     let refund_response = CONNECTOR
-        .make_payment_and_refund(payment_method_details(), None, get_default_payment_info())
+        .make_payment_and_refund(
+            payment_method_details(100),
+            None,
```

Comment:
pass full amount for refund

Reply:
It will by default pick 100, Rest at every place it is handled